README.md

```diff
@@ -1,15 +1,12 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
 
-> Description the project.
-
+Hello World
 
 ## Built With
 
-- Major languages
+- html
 - Frameworks
-- Technologies used
+- vs code, github, gitbash
 
 ## Live Demo (if available)
```

Comment:
[OPTIONAL]  I kindly suggest you to remove this section because you haven't a demo link